src/components/BadgeBuilderDialog.vue

```diff
@@ -0,0 +1,152 @@
+<template>
```

Comment:
Please rename this component `StatusPageSVGBadgeDialog.vue` to reflect the tight focus of this component.